core/__tests__/modules/configWriter.ts

```diff
@@ -806,7 +806,8 @@ describe("modules/configWriter", () => {
     test("properties should only humanize their ID if it matches default pattern", async () => {
       let property: Property = await helper.factories.property(
         source,
-        { key: faker.lorem.word() },
+        // @ts-expect-error can be removed once faker.unique types are fixed (https://github.com/faker-js/faker/pull/333)
+        { key: faker.unique(faker.lorem.word) },
```

Comment:
You can also just use something that you will be sure is unique: 
* `${new Date().getTime()}-${Math.random()}`
* uuid.v4()